components/webdriver_server/actions.rs

```diff
@@ -738,4 +684,93 @@ impl Handler {
             None => Err(ErrorStatus::UnknownError),
         }
     }
+
+    /// <https://w3c.github.io/webdriver/#dfn-extract-an-action-sequence>
+    pub(crate) fn extract_an_action_sequence(&self, params: ActionsParameters) -> ActionsByTick {
+        // Step 1. Let actions be the result of getting a property named "actions" from parameters.
+        // Step 2 (ignored). If actions is not a list, return an error with status InvalidArgument.
```

Comment:
Better to say `params` is already validated in earlier steps so that people don't wonder why it can be ignored.